app/screens/contact/contact.js

```diff
@@ -1,15 +1,140 @@
 import React, { Component } from 'react';
 import { 
-  View, Text
+  View,
+  Text,
+  Image,
+  SectionList
  } from 'react-native';
-import style from './styles';
+import Swiper from 'react-native-swiper'; 
+import Search from 'react-native-search-box';
+import ParallaxScrollView from 'react-native-parallax-scroll-view';
 
+import colors from 'App/config/colors';
+import ContactCell from './contactCell';
+import screens from 'App/constants/screens';
+import { getWidth, getHeight } from 'App/utils/dimension';
+import style, { AVATAR_SIZE, STICKY_HEADER_HEIGHT, DOT_MARGIN, PARALLAX_HEADER_HEIGHT } from './styles';
+
+const DEPARTMENT_LIST = ["ios", "android", "java", "php", "reactnative", "pm"];
```

Comment:
We'll need to move this into a separate constants file when removing dummy data.

Reply:
This is also a dummy data, These kinda data comes from the api